RPIPs/RPIP-59.md

```diff
@@ -31,9 +31,9 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 ### Deposit queue specification
 ETH from the deposit pool SHALL be matched with validator deposits from queues as follows:
 - There SHALL be a `standard_queue`
-  - When adding a validator, users MAY place their deposit on the `standard_queue`  
+  - When adding a validator, users SHALL be able to place their deposit on the `standard_queue`  
```

Comment:
```suggestion
  - When adding a validator, users SHALL be able to place their deposit in the `standard_queue`  
```

Reply:
Minor note. On the queue? Is this common wording? In the UK I would say 'in the queue'. Clearly if there is any authority here on the proper queuing terminology, it belongs on this side of the atlantic. 

Yes. This comment is mostly here as a joke, and you can remove without actioning.